tests/test_models/test_task_modules/test_estimators/test_flops_params.py

```diff
@@ -127,6 +128,15 @@ def test_estimate(self) -> None:
         self.assertGreater(flops_count, 0)
```

Comment:
add more strict check constraints for `::test_estimate` UTs, not just greater than 0.

Reply:
Done.